source/macos/AdaptiveCards/AdaptiveCards/Renderers/ColumnRenderer.swift

```diff
@@ -22,32 +22,30 @@ class ColumnRenderer: BaseCardElementRendererProtocol {
             topSpacingView = view
         }
         
-        for (index, item) in column.getItems().enumerated() {
-            let renderer = RendererManager.shared.renderer(for: item.getType())
-            let view = renderer.render(element: item, with: hostConfig, style: style, rootView: rootView, parentView: columnView, inputs: [], config: config)
+        for (index, element) in column.getItems().enumerated() {
+            let isFirstElement = index == 0
+            let renderer = RendererManager.shared.renderer(for: element.getType())
+            let view = renderer.render(element: element, with: hostConfig, style: style, rootView: rootView, parentView: columnView, inputs: [], config: config)
             columnView.configureColumnProperties(for: view)
-            let viewWithInheritedProperties = BaseCardElementRenderer.shared.updateView(view: view, element: item, rootView: rootView, style: style, hostConfig: hostConfig, config: config, isfirstElement: index == 0)
-            columnView.addArrangedSubview(viewWithInheritedProperties)
-            BaseCardElementRenderer.shared.configBleed(collectionView: view, parentView: columnView, with: hostConfig, element: item, parentElement: column)
+            BaseCardElementRenderer.shared.updateLayoutForSeparatorAndAlignment(view: view, element: element, parentView: columnView, rootView: rootView, style: style, hostConfig: hostConfig, config: config, isfirstElement: isFirstElement)
+            BaseCardElementRenderer.shared.configBleed(collectionView: view, parentView: columnView, with: hostConfig, element: element, parentElement: column)
         }
         
+        columnView.configureLayout(column.getVerticalContentAlignment(), minHeight: column.getMinHeight() ?? 0, heightType: column.getHeight(), type: .column)
```

Comment:
Do we need to set min height for all elements?

Reply:
this function define layout and visibility(visibility will add upcoming commit) not just for min height. and if there scenario where column has min height it will add support otherwise ignore it.